tkey/src/main/java/com/web3auth/tkey/ThresholdKey/Common/GetTSSPubKeyResult.java

```diff
@@ -20,6 +20,11 @@ public Point(String x, String y) {
     public Point publicKey;
     public List<Integer> nodeIndexes;
 
+    /**
+     * Instantiate a GetTSSPubKeyResult object using the public key and nodeIndexes.
+     * @param publicKey The publicKey from torus utils getPublicAddress
+     * @param nodeIndexes The node indexes from torus utils nodes data
+     */
     public GetTSSPubKeyResult(Point publicKey, List<Integer> nodeIndexes) {
```

Comment:
Hmm, probably could be better named
```suggestion
    public TSSPubKeyResult(Point publicKey, List<Integer> nodeIndexes) {
```